README.md

```diff
@@ -89,6 +89,20 @@ This config option controls the export file format for each Sphinx builder. It
 accepts a dictionary mapping builder names to image formats. Accepted values for
 the latter are `"png"`, `"jpg"`, or `"svg"`.
 
+### Output directory
+- *Formal Name*: `drawio_export_directory`
+- *Default Value*: `None`
+
+This config option controls where the generated files are exported to.
+Both relative(relative to the current working directory) and absolute paths
+are supported.
+
+If not set all files are generated into the builders output directory
+e.g. `_build/html/.drawio/*`
```

Comment:
Could we use `/drawio`? `.drawio` paths will be hidden on Linux, which makes this option undesirable.

Reply:
I agree - working on linux here - I just kept this as it was (thought this might make sense to anyone ;) )

Reply:
I was in favor of ".drawio" since the default doctrees dir is ".doctrees", but I don't have strong feelings about it.